logging/fb/fluent-bit_config.configmap_opensearch.yaml

```diff
@@ -1038,7 +1038,12 @@ data:
        storage.sync normal
        storage.backlog.mem_limit 5M
        storage.metrics On
+
+    @INCLUDE viya-tracing.conf
```

Comment:
Fluent Bit doc says "_The @INCLUDE command only works at top-left level of the configuration line, it cannot be used inside sections._"  Based on that, I wonder if this would be better placed at the top rather than at the bottom.  Wouldn't impact functionality but would minimize chances of it getting "lost" after 1000 lines of log processing logic.  On the other hand, I suppose we plan on breaking this configuration into more manageable blocks/modules at some point; so maybe not worth changing for now.

Reply:
I have no problem moving this to the top but just to clarify, the "top" level the doc is referring is in reference to the indentation, meaning it can't be placed under `filter` or one of the other sections. But sure, def would make it more readable if it sits at the top

Reply:
Understood.  My concern was mostly centered on making it obvious when we're including another file.  I also wonder about inclusion order and how duplicate/conflicting values are handled (i.e. are included files processed first and then the remaining content?  Or, are included files processed as they are encountered?).  We may want/need to dig into those messy details as we "modularize".